src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,60 @@
+import { useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { useContext } from 'react';
+import { Person } from '../../types';
+import { PeopleContext } from '../../PeopleContext';
+
+type Props = {
+  person: Person
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { personSlug } = useParams();
+  const {
+    findPersonFather,
+    findPersonMother,
+    people,
+  } = useContext(PeopleContext);
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn('',
+        { 'has-background-warning': personSlug === person.slug })}
+    >
+      <td>
+        <a
+          href={`#/people/${person.slug}`}
+          className={cn('', { 'has-text-danger': person.sex === 'f' })}
+        >
+          {person.name}
+        </a>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      {findPersonMother(people, person)
```

Comment:
let's save the result of this function in some variable to remove redundant call